<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>581</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Light</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>KnowClass</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>9</y>
      <width>731</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tree_tab">
     <attribute name="title">
      <string>Дерево</string>
     </attribute>
     <widget class="QLabel" name="tree_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>721</width>
        <height>501</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Emoji</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Дерево&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>0</y>
        <width>16</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="matrix_tab">
     <attribute name="title">
      <string>Матрица</string>
     </attribute>
     <widget class="QTableView" name="matrix_tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Консультация</string>
     </attribute>
     <widget class="QPushButton" name="beginConsultingButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>0</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать консультацию</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="open_menu">
     <property name="title">
      <string>Открыть</string>
     </property>
     <addaction name="separator"/>
     <addaction name="tree_open_menu"/>
     <addaction name="matrix_open_menu"/>
    </widget>
    <addaction name="open_menu"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_menu"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="about_menu">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="tree_open_menu">
   <property name="text">
    <string>Дерево</string>
   </property>
  </action>
  <action name="matrix_open_menu">
   <property name="text">
    <string>Матрица</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
